build_system/seqan3-config.cmake

```diff
@@ -158,6 +148,39 @@ macro (seqan3_config_error text)
     endif ()
 endmacro ()
 
+# ----------------------------------------------------------------------------
+# Detect if we are a clone of repository and if yes auto-add submodules
+# ----------------------------------------------------------------------------
+
+if (EXISTS "${CMAKE_CURRENT_LIST_DIR}/../build_system")
+    message (STATUS "  Detected as running from a repository checkout…")
+
+    if (IS_DIRECTORY "${CMAKE_CURRENT_LIST_DIR}/../include")
```

Comment:
It could be safer to check if `.../include/seqan3/version.hpp` exists. But maybe this will be detected later already?

Reply:
yep